### DISCUSSION TITLE
Disable Features Serverside

### DISCUSSION THREAD

**Discussion by Zoriot** [maruohon/tweakeroo#286] [Q&A] [answered]
Is there a way to disable Features like the Free Cam Server Side? So No User can use it.
Would be very nice if this is possible.

**Comment by maruohon** [comment, marked as answer]
Not currently, but this is possible in the rewritten code base I'm working on in 1.12.2, thanks to various malilib changes and improvements. So once all that new code is ready to get ported, then you will be able to set a special (invisible to normal clients) line in the server MOTD to apply config locks/overrides to any mods using the malilib config system.

**Reply by flooorian1** [reply]
Hi, is there anything new about this? Is it currently possible to disable specific things for players on a server?

**Reply by maruohon** [reply]
It's still only in the 1.12.2 main development code base.

**Reply by circularsprojects** [reply]
Are there any new updates on this?

**Reply by maruohon** [reply]
Unfortunately not really. It's still only in the main development code in 1.12.2. My time and motivation to work on the mods has diminished significantly in the past couple of years, so there has been varying levels of progress on stuff, and lately it has been pretty near zero progress on any mod stuff. malilib is nearly ready for releases, but the other mods still need some work before I'd like to port them from 1.12.2 to the modern MC versions.
